pkg/skaffold/build/local/docker.go

```diff
@@ -47,9 +48,15 @@ func (b *Builder) buildDocker(ctx context.Context, out io.Writer, a *latest.Arti
 	var imageID string
 
 	if b.local.UseDockerCLI || b.local.UseBuildkit {
-		imageID, err = b.dockerCLIBuild(ctx, out, a.Workspace, a.ArtifactType.DockerArtifact, tag)
+		imageID, err = b.dockerCLIBuild(ctx, out, a.Workspace, a.ArtifactType.DockerArtifact, a.Dependencies, tag)
```

Comment:
I think you should pass same same BuildOptions as you do below.

Reply:
will refactor

Reply:
done.